lib/logger/lib/logger/handler.ex

```diff
@@ -91,7 +91,11 @@ defmodule Logger.Handler do
 
           {message, %{gl: gl} = metadata} ->
             timestamp = Map.get_lazy(metadata, :time, fn -> :os.system_time(:microsecond) end)
-            metadata = erlang_metadata_to_elixir_metadata(metadata)
+
+            metadata =
+              erlang_metadata_to_elixir_metadata(metadata)
+              |> Keyword.put(:erl_level, erl_level)
```

Comment:
```suggestion
              [erl_level: erl_level] ++ erlang_metadata_to_elixir_metadata(metadata)
```

Reply:
thanks for suggestion!

Reply:
P.S. thanks again for suggestion but when I read it the merge was already done, thanks again and sorry for delay

Reply:
It was my bad! I usually apply the suggestion myself but I forgot! :D I will fix it locally, thanks!